packages/lambdas/src/document-uploader.ts

```diff
@@ -40,10 +42,15 @@ export const handler = async (event: S3Event) => {
         });
       }
     } catch (error) {
-      console.log("Error in documentUploaderHandler", error);
-      capture.error(error, {
-        extra: { context: `documentUploaderHandler`, sourceBucket, sourceKey, error },
+      const message = "Error in documentUploaderHandler";
+      console.log(message, error);
+      capture.setExtra({
+        context: `documentUploaderHandler`,
+        sourceBucket,
+        sourceKey,
+        error,
```

Comment:
Can you check if we can skip sending error here since we're passing it to MetriportError, please? It'd be good if you could share the link to the Sentry event so we can confirm the stack/original error is available there. :)

Reply:
done